extension/intents/tabs/tabs.toml

```diff
@@ -401,14 +401,23 @@ description = "Find on page content"
 match = """
   find [query] on (this |) (page | tab)
   search (this |) (page | tab) for [query]
+  [query]
```

Comment:
This should be a new intent. Also we definitely don't want a match that catches everything!

The implementation is going to look a lot like the `navigation.followLink` intent (and can probably share code with it). As proposed "sign in" will just click a link matching `sign in | login | log in` (note unlike followLink it doesn't matter as much if the link is visible on the screen – followLink prefers links that are visible on the screen)